lib/pbench/test/unit/agent/task/test_copy_result_tb.py

```diff
@@ -29,9 +30,9 @@ def test_copy_tar(self, valid_config):
             status=200,
         )
         try:
-            crt = CopyResultTb("controller", tarball, self.config, self.logger)
-            crt.copy_result_tb()
-        except SystemExit:
+            crt = CopyResultTb("ctrl", tarball, self.config, self.logger)
+            crt.copy_result_tb("token")
+        except FileUploadError:
             assert False
         else:
             assert True
```

Comment:
The try/except shouldn't be necessary here; an unexpected exception (without a `with pytest.raises()`) will fail the test, and you're not providing any additional diagnostic information here.

Reply:
Done